src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1280,7 +1283,10 @@ function ($a) {
                             ->defaultNull()
                             ->info('Transport name to send failed messages to (after all retries have failed).')
                         ->end()
-                        ->scalarNode('default_bus')->defaultNull()->end()
+                        ->scalarNode('default_bus')
+                            ->defaultNull()
+                            ->info('Bus name that will be used for autowiring MessageBusInterface. When there is only a single bus this happens automatically. With multiple buses you can set this option or use named autowiring based on the bus name or explicit wiring.')
```

Comment:
```suggestion
                            ->info('Bus name that will be used for autowiring MessageBusInterface. When there is only a single bus this happens automatically. With multiple buses, you can set this option or use named autowiring based on the bus name (or explicit wiring).')
```

Mostly, the last part of the sentence was confusing because it was "you can do X, or Y or Z" when really it's mean to read like "you can do X or Y (and Y is really 2 things). Hopefully this is clearer - it's minor regardless.